### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by yogitasaini**
find small bug in bubblesort.java.
please review it.

### CODE REVIEW

**Comment by ritvij14** [comment]
@yogitasaini did you try running the code before making the changes? Was it actually not sorting correctly? Can you please show an example? Because I cannot see where exactly is the `sorted` variable being used.

Please give an example of what the bug was initially and how does your addition fix it.

**Comment by yogitasaini** [comment]
It will not happen every time that our array runs n time. Suppose that if the array is sorted in first iterate then I put a check for those cases so it not need to iterate again in the loop

File: Java/sort/BubbleSort.java [hunk 1]
```diff
@@ -3,12 +3,14 @@ static void bubbleSort(int[] arr) {
...
          for(int i=0; i < n; i++){  
+            boolean sorted = true;
              for(int j=1; j < (n-i); j++){
                  if(arr[j-1] > arr[j]){
                      //swap elements
                      temp = arr[j-1];
                      arr[j-1] = arr[j];
                      arr[j] = temp;
+                    sorted = false;
                  }
```
Comment (ritvij14) [inline Java/sort/BubbleSort.java:15]: @yogitasaini then in that case, shouldn't there be a check here in this line that checks for the value of `sorted`? If it's true, then there is no need to execute the loop anymore, so the loop should stop? I think you should add a check here.

**Review by ritvij14 — CHANGES_REQUESTED** [review]

**Comment by yogitasaini** [comment]
Hey, @HarshCasper @ankitaggarwal23 @anushkrishnav @siddharth25pandey ,please do review my pr
Thank you.

**Review by ritvij14 — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**